### PR TITLE
Student demographics added, fixtures

### PR DESCRIPTION
**Pull request by pjsier**
Model for Race/Ethnicity added, associated with Students on a many to many and school name and school type added as attributes of Student.

Also, fixtures for race added, and tweak to age accepting an optional argument to generate date in the past added

### CODE REVIEW

**Comment by pjsier** [comment]
Removed age edits to put in a separate PR

**Comment by pjsier** [comment]
Ages tweak fully fixed, added back into PR

File: coderdojochi/forms.py [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
 
 import html5.forms.widgets as html5_widgets
 
+school_type_choices = (
+("Public", "Public"),
+("Charter", "Charter"),
+("Private", "Private"),
+("Homeschool", "Homeschool")
+)
+
```
Comment (bkellgren) [inline coderdojochi/forms.py]: Super small comment but I am going to be implementing  PEP8 standards throughout the codebase here shortly so you'd want to indent the options in this tuple.  If your using Sublime, [this](https://github.com/SublimeLinter/SublimeLinter-pep8) is a great tool.

**Comment by bkellgren** [comment]
:+1: Looks great!

**Comment by pjsier** [comment]
Just updated! And good to know about PEP8, thanks!

**Pull request closed without merging**